### ISSUE TITLE
AccessibilityRole should be configurable

### ISSUE THREAD

**Issue by felire** [satya164/react-native-tab-view#1093]
Not too much, but I think that the accessibility props should be configurable. Or do you think they should be like it is now? I could make a pr if you are agree with the change

**Comment by felire** [comment]
I've already done the pr [https://github.com/satya164/react-native-tab-view/pull/1095](https://github.com/satya164/react-native-tab-view/pull/1095)
